test/cask-api-test.el

```diff
@@ -630,6 +630,19 @@
       (cask-install bundle)
       (should (equal orig-load-path load-path)))))
 
+(ert-deftest cask-install-test/pin-archive ()
+  (if  (not (boundp 'package-pinned-packages))
+    (princ "`package-pinned-packages' is not defined, we need Emacs v24.4+ to test the pinning feature.\n")
+    (cask-test/with-bundle
+     '((source localhost)
+       (depends-on "package-a" "0.0.1" :archive "localhost"))
```

Comment:
You're talking about `"localhost"` in `(depends-on "package-a" "0.0.1" :archive "localhost")` ?
The doc of `package-pinned-packages` says (emphasis is mine):
> Each element of the alist has the form (PACKAGE . ARCHIVE), where:
    PACKAGE is a symbol representing a package
    ARCHIVE is a **string** representing an archive (it should be the car of
   an element in `package-archives', e.g. "gnu").